CSharpier.sln

```diff
@@ -64,6 +65,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CSharpier.Tests.Generators"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CSharpier.Generators", "Src\CSharpier.Generators\CSharpier.Generators.csproj", "{D846F494-57FC-4315-A2A8-357343175D2E}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CSharpier.CLI", "src\CSharpier.CLI\CSharpier.CLI.csproj", "{F2BF106F-BB0A-44AC-8249-91BC53FFC83A}"
```

Comment:
So, the .net style guide on capitalization says to treat [acronyms longer than 2 letters as regular words](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers).

According to that, this project should be `CSharpier.Cli`.
That is, if you so happen to chose that style guide for this project :wink: